test/runtests.jl

```diff
@@ -2011,3 +2011,12 @@ end
     @test_throws ArgumentError lm(@formula(OptDen ~ Carb), form; method=:pr)
     @test_throws ArgumentError glm(@formula(OptDen ~ Carb), form, Normal(); method=:pr)
 end
+
+@testset "[G]VIF" begin
+    duncan = RDatasets.dataset("car", "Duncan")
+    lm1 = lm(@formula(Prestige ~ 1 + Income + Education), duncan)
+    @test termnames(lm1)[2] == coefnames(lm1)
```

Comment:
1. do we have tests when `coefnames` and `termnames` differ?
2. do we have a decision what should be done in the case of `lm(X, y)` (i.e. model fitted without formula, it still prints variable names as `x1` etc.)

Reply:
1. This falls back to StatsModels -- the test there is just making sure we've successfully imported and exported the symbol.
2. on `master`,`termnames` will error based on there being no formula (`formula` will return `nothing`).

Reply:
I think `termnames` should not be defined if there is no formula -- there are only Terms when there is a formula.

Reply:
So what should a user do to perform VIF analysis for the `model - lm(X, y)` case?

Reply:
`vif` works, but not `gvif`. So I think they can still do `vif`. If they're able to construct a model matrix directly for something with non trivial contrast coding, then they could probably also do adapt the `gvif` source to extract the correct columns.